### PR TITLE
Current event

### PR DESCRIPTION
**Pull request by jake-shasteen**

### CODE REVIEW

File: src/components/Banner.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 
 const Banner = () => (
   <div id="banner-container">
-    <div className="banner">
+    <div className="center banner">
       <p>
         See what events are happening around you.
         Check out what's happening at local venues - find your Jamboree.
```
Comment (kathy-ems) [inline src/components/Banner.js:8]: This is highlighted red and I don't know why.

File: src/components/Event.js [hunk 2]
```diff
@@ -6,9 +6,10 @@ class Event extends Component {
   }
 
   render() {
+    let className = this.props.currentEvent === this.props.id ? 'center event currentEvent' : 'event';
```
Comment (kathy-ems) [inline src/components/Event.js:9]: Clever idea to put the class name inside the code to dynamically set it based on whether it's the current event.

**Comment by kathy-ems** [comment]
- The search appears to be broken for common places like "nyc". This may be because this branch doesn't have the latest upstream bug fix for search.
- Also, clicking many times on the pins, eventually, the pins stop being clickable and re-centering. The one it seems to get stuck on is the event at fort mason.

File: src/components/Banner.js (hunk 1 above)
Comment (jake-shasteen) [inline src/components/Banner.js:8]: No idea why either! It was red in the previous version as well, it looks like? Some weird parsing in github's highlighter?

**Comment by jake-shasteen** [comment]
- Search for 'nyc' seems fine when I make a new branch from current master and pull this PR down. I'll get with you in person to see this in action.
- The bug you mention regarding clicking pins has a ticket in Trello, but is not related to these changes. I believe it has to do with the video id not being defined ( maybe no video could be found for a particular search?)

**Pull request merged**